pkg/utils/reader.go

```diff
@@ -72,6 +90,98 @@ func InitFileReader() *FileReader {
 	return &FileReader{}
 }
 
+type msg struct {
+	Msg        string
+	City       string
+	Country    string
+	Latency    int
+	Batch      string
+	FirstName  string
+	LastName   string
+	Gender     string
+	Hostname   string
+	HTTPStatus int
+	Hobby      string
```

Comment:
nit: we should not use the "person" tags here, @sunitakawane we were supposed to use tags that we see in aws env, like : dc, pod, region, az, hostname, etc...